Sources/Networking/Auth/OAuthClient.swift

```diff
@@ -237,11 +259,12 @@ final public class DefaultOAuthClient: OAuthClient {
                 return refreshedTokens
             } catch OAuthServiceError.authAPIError(let code) where code == OAuthRequest.BodyErrorCode.invalidTokenRequest {
                 Logger.OAuthClient.error("Failed to refresh token: invalidTokenRequest")
-                throw OAuthClientError.refreshTokenExpired
+                throw OAuthClientError.invalidTokenRequest
```

Comment:
here I maintain a similar error to the one sent by the API. `.invalidTokenRequest` can happen for other reasons than a dead token, for example when we try to use a staging token in production.